src/lib/types/capi.ts

```diff
@@ -0,0 +1,37 @@
+export type Source = {
+	minWidth: string;
+	sizes: string;
+	hidpiSrcset: string;
+	lodpiSrcset: string;
+};
+
+export type Branding = {
+	brandingType: {
+		name: string;
+	};
```

Comment:
Not allowed to commit suggestions any longer 😢 – can you, @chrislomaxjones ?

```suggestion
/**
 * Best effort Scala conversion from [Branding.scala](https://github.com/guardian/commercial-shared/blob/a692e8b2eba6e79eeeb666e5594f2193663f6514/src/main/scala/com/gu/commercial/branding/Branding.scala#L7-L22)
 */
export type Branding = {
	brandingType: {
		name: string;
	};
```

Reply:
Of course!